### ISSUE TITLE
Excalidraw Slow on Samsung Tablets

### ISSUE THREAD

**Issue by devinat1** [excalidraw/excalidraw#5946]
After just a couple of drawing strokes, I notice a significant decrease in performance for excalidraw. Am using a samsung S7+ tablet.
I end up having to create multiple drawings to be able to draw a little bit in each one of them, which is quite cumbersome.

**Comment by nippyseven** [comment]
Yes I am experiencing the same issue on my galaxy tab s6 lite (2022). I understand Excalidraw is not designed for pen based writing but this lag (which is consistent across all mobile devices) makes the plugin borderline unusable for writing on such devices . I hope this can be optimised in the future as it would enable more students like myself to be able to rely exclusively on excalidraw+obsidian as a note taking software. I usually would work around this lag by using my tablet as a second monitor for my PC (ryzen 5 3600 + gtx 1080) but i spilt coffee on it today. :/
